### PR TITLE
Update troubleshooting-known-issues.md

### PR DESCRIPTION
**Pull request by anunesms**
Adding known scenario of STS in sensor service fails to start and moving section to top of the page.

### CODE REVIEW

**Comment by anunesms** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by Court72** [comment]
@batamig

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: ATPDocs/troubleshooting-known-issues.md [hunk 1]
```diff
@@ -9,6 +9,66 @@ ms.topic: troubleshooting
...
+Set-ADServiceAccount mdiSvc01 -PrincipalsAllowedToRetrieveManagedPassword $group
+```
+
+If the domain controller or security group is already added, but you're still seeing the error, you can try the following steps:
+
+- **Option 1**: Reboot the server to sync the recent changes
+- **Option 2**:
+    1. Set the **AATPSensor** and **AATPSensorUpdater** services to Disabled
+    1. Stop the **AATPSensor** and **AATPSensorUpdater** services
+    1. Cache service account to server using the command: `Install-ADServiceAccount gMSA_AccountName`
```
Comment (martin77s) [inline ATPDocs/troubleshooting-known-issues.md:55]: @anunesms 
Let's remove option 2 from here. 
There's no need to install the service account on the server. It's relevant only for MSAs and not gMSAs.

File: ATPDocs/troubleshooting-known-issues.md [hunk 2]
```diff
@@ -9,6 +9,66 @@ ms.topic: troubleshooting
 
 This article describes how to troubleshoot known issues in Microsoft Defender for Identity. 
 
+## Sensor service fails to start
+
+**Sensor log entries:**
+
+`Warn DirectoryServicesClient CreateLdapConnectionAsync failed to retrieve group managed service account password. [DomainControllerDnsName=DC1.CONTOSO.LOCAL Domain=contoso.local UserName=AATP_gMSA]`
```
Comment (martin77s) [inline ATPDocs/troubleshooting-known-issues.md:16]: @anunesms 
Let's change the username to mdiSvc01 so it matches other pages.

File: ATPDocs/troubleshooting-known-issues.md [hunk 3]
```diff
@@ -9,6 +9,66 @@ ms.topic: troubleshooting
...
+
+### Cause 1
+
+The domain controller hasn't been given rights to access the password of the gMSA account.
+
+**Resolution 1:**
+
+Verify that the domain controller has been given rights to access the password. You should have a Security Group in Active Directory that contains the domain controller(s), AD FS / AD CS server(s) and standalone sensors computer accounts included. If this doesn't exist, we recommend that you create one.
```
Comment (martin77s) [inline ATPDocs/troubleshooting-known-issues.md:24]: @anunesms 
Let's add Entra Connect

File: ATPDocs/troubleshooting-known-issues.md [hunk 4]
```diff
@@ -9,6 +9,66 @@ ms.topic: troubleshooting
...
+- **Option 2**:
+    1. Set the **AATPSensor** and **AATPSensorUpdater** services to Disabled
+    1. Stop the **AATPSensor** and **AATPSensorUpdater** services
+    1. Cache service account to server using the command: `Install-ADServiceAccount gMSA_AccountName`
+    1. Set the **AATPSensor** and **AATPSensorUpdater** services to Automatic
+    1. Start the **AATPSensorUpdater** service
+ 
+### Cause 2
+
+Due to a known scenario related to Secure Time Seeding, the GMSA attribute PasswordLastSet can be set to a future date, causing the sensor to be unable to start.
```
Comment (martin77s) [inline ATPDocs/troubleshooting-known-issues.md:61]: @anunesms 
Rename GMSA to gMSA

File: ATPDocs/troubleshooting-known-issues.md [hunk 5]
```diff
@@ -9,6 +9,66 @@ ms.topic: troubleshooting
...
+    1. Set the **AATPSensor** and **AATPSensorUpdater** services to Automatic
+    1. Start the **AATPSensorUpdater** service
+ 
+### Cause 2
+
+Due to a known scenario related to Secure Time Seeding, the GMSA attribute PasswordLastSet can be set to a future date, causing the sensor to be unable to start. 
+The following command can be used to confirm if the GMSA account falls in the scenario:
+
+```powershell
+Get-ADServiceAccount gmsa-name -Properties *
```
Comment (martin77s) [inline ATPDocs/troubleshooting-known-issues.md:65]: @anunesms 
Rename gmsa-name to mdiSvc01

**Comment by batamig** [comment]
Hi @anunesms and thanks for your contribution. As internal to Microsoft, you can access the private repo, where we can make suggestions / changes, and the system will run validations and give you a staging page to preview.
Given the automation available internally, we encourage private repo usage when available. I'm going to close this PR for now - but no worries, it doesn't get deleted, and your changes are still here.
#please-close

Please recreate this PR in the private repo and include Martin's feedback. I'll contact you offline with more details.

**Pull request closed without merging**